.changeset/slimy-windows-grab.md

```diff
@@ -0,0 +1,12 @@
+---
+'@clerk/backend': minor
+---
+
+Deprecate `createSMSMessage` and `SMSMessageApi` from `clerkClient`.
+
+The equivalent `/sms_messages` Backend API endpoint will also be dropped in the future, since this feature will no longer be available for new instances.
```

Comment:
```suggestion
The `/sms_messages` Backend API endpoint will also be dropped in the future since this feature will no longer be available for new Clerk instances.
```

Reply:
@SokratisVidros I had enabled auto merge from yesterday and did not get a chance to update the copyright 😓

Reply:
Its updaded in https://github.com/clerk/javascript/pull/2184 which is for v4 release